app/components/branch-row.js

```diff
@@ -41,7 +41,8 @@ export default Ember.Component.extend({
         };
       }
       let path = `${apiEndpoint}/v3/repo/${repoId}/builds`;
-      let params = `?branch.name=${branchName}&limit=5&build.event_type=push,api,cron`;
+      // eslint-disable-next-line
+      let params = `?branch.name=${branchName}&limit=5&build.event_type=push,api,cron&sort_by=finished_at:desc`;
```

Comment:
This doesn't seem like something that belongs to this PR, meaning that this changes the current functionality of how the branch page looks like